3p/iframe-messaging-client.js

```diff
@@ -103,11 +103,19 @@ export class IframeMessagingClient {
     });
   }
 
+  /**
+   * @param {!Window} win
+   */
   setHostWindow(win) {
+    /** @private {!Window} */
```

Comment:
You've mutated the hidden class here, which deoptimizes property access on the object. You've got to set each property in the constructor, even if it's to `null`.